drivers/CoreLED/source/CoreLED.cpp

```diff
@@ -25,3 +25,17 @@ void CoreLED::showColor()
 	// TODO (@hugo) - frame end must be generated depending on the size of the led strip
 	_spi.write(frame::end);
 }
+
+void CoreLED::showColor()
+{
+	_is_on = true;
+	showColor(_color);
```

Comment:
généralement on met le changement de status à la fin des actions. ça permet que si on ajoute des checks après, le status ne change pas de valeur alors qu'un check fail. tu veux être sûr que tout ce qui a doit avoir lieu a bien lieu.

```suggestion
	showColor(_color);
	_is_on = true;
```